src/matchers/toBeOdd/index.js

```diff
@@ -0,0 +1,25 @@
+import { matcherHint, printExpected, printReceived } from 'jest-matcher-utils';
+
+import predicate from './predicate';
+
+const passMessage = received => () =>
+  matcherHint('.not.toBeOdd', 'received', '') + '\n\n' + 'Expected value to not be odd received:\n' + `  ${received}`;
```

Comment:
The `received` value should be wrapped in the printReceived function. `${printReceived(received)};`